docs/angular.md

```diff
@@ -25,10 +27,10 @@ _Install Angular CLI_
 npm install -g @angular/cli
 ```
 
-### Starting A New Project
+### 2) Starting A New Project
 
 ```bash 
-ng new PROJECT-NAME --service-worker true --routing true --spec false --prefix COMPONENT-PREFIX
+ng new PROJECT-NAME --service-worker true --routing true --spec false --prefix pod
```

Comment:
If this is meant to be generic, and it is, then COMPONENT_PREFIX is more appropriate

Reply:
fixed